src/fprime_gds/executables/cli.py

```diff
@@ -651,6 +651,14 @@ def get_arguments(self) -> Dict[Tuple[str, ...], Dict[str, Any]]:
                     "type": str,
                     "help": "Path to packet specification.",
```

Comment:
We should call out on this one that this is now deprecated, preferring the JSON way.

Reply:
Will do!